app/components/blacklight/search/facet_filters_component.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Blacklight::Search
+  class FacetFiltersComponent < Blacklight::Component
+    # @param [Blacklight::FacetFieldPresenter] presenter
+    def initialize(presenter:, classes: 'facet-filters card card-body bg-light p-3 mb-3 border-0')
+      @presenter = presenter
+      @classes = classes
+    end
+
+    def facet
+      @presenter.facet_field
+    end
+
+    attr_reader :classes, :presenter
+
+    delegate :display_facet, to: :presenter
+
+    def render?
+      facet.suggest != false || render_index_navigation?
```

Comment:
```suggestion
      facet.suggest != false || render_index_navigation?
      facet.suggest.present? || render_index_navigation?
```

Reply:
It seems like FacetSuggestInput has the possibility of facet being nil. Also shouldn't the render? statement be removed from FacetSuggestInput if it is getting moved?

Reply:
I don't know. I just know what the current code has: https://github.com/projectblacklight/blacklight/blob/4a9f4263a534e573718d99c68cfeb7e8771badaa/app/components/blacklight/search/facet_suggest_input.rb#L16

Reply:
I thought the current code was wrong, so I fixed it.

Reply:
@jcoyne can we still use facet.suggest.present? instead of != false or are we fine with facet.suggest = '' and we want that?

Reply:
Facet suggest is a boolean field.  We'd never expect it to have strings.

Reply:
Then shouldn't the line be facet.suggest if it is a boolean. I am a little confused why there is a check for it against false

Reply:
No, because it could also be nil.

Reply:
so it should render if facet.suggest == nil?

Reply:
Right. it has to match this: https://github.com/projectblacklight/blacklight/blob/main/app/components/blacklight/facets/suggest_component.rb#L17-L18